viz_scripts/scaffolding.py

```diff
@@ -110,20 +110,28 @@ def energy_intensity(df,df1,distance,col1,col2):
     distance = distance in meters
     col1 = Replaced_mode
     col2= Mode_confirm
+
     """
     df['distance_miles']= df[distance]*0.00062 #meters to miles
     df1 = df1.copy()
     df1[col1] = df1['mode']
     dic_ei_factor = dict(zip(df1[col1],df1['energy_intensity_factor']))
     dic_CO2_factor = dict(zip(df1[col1],df1['CO2_factor']))
+    dic_ei_trip = dict(zip(df1[col1],df1['(kWH)/trip']))
+    
     df['ei_'+col1] = df[col1].map(dic_ei_factor)
     df['CO2_'+col1] = df[col1].map(dic_CO2_factor)
+    df['ei_trip_'+col1] = df[col1].map(dic_ei_trip)
     
+      
     df1[col2] = df1[col1]
     dic_ei_factor = dict(zip(df1[col2],df1['energy_intensity_factor']))
+    dic_ei_trip = dict(zip(df1[col2],df1['(kWH)/trip']))
     dic_CO2_factor = dict(zip(df1[col2],df1['CO2_factor']))
     df['ei_'+col2] = df[col2].map(dic_ei_factor)
     df['CO2_'+col2] = df[col2].map(dic_CO2_factor)
+    df['ei_trip_'+col2] = df[col2].map(dic_ei_trip)
```

Comment:
@lboyce maybe I am missing something obvious, but where are you using the newly computed `ei_trip_` column? I looked through the file and couldn't see it.

Reply:
nvm I figured it out.